warehouse/locale/Makefile

```diff
@@ -0,0 +1,46 @@
+GITHUB_ACTIONS := $(shell echo "$${GITHUB_ACTIONS:-false}")
+
+compile-pot:
+	cd ../../; \
+	PYTHONPATH=$(PWD) pybabel extract \
+		-F babel.cfg \
+		--omit-header \
+		--output="warehouse/locale/messages.pot" \
+		warehouse
+
+init-po:
+	cd ../../; \
+	pybabel init \
+		--input-file="warehouse/locale/messages.pot" \
+		--output-dir="warehouse/locale/" \
+		--locale="$(L)"
+
+update-po:
+	cd ../../; \
+	pybabel update \
+		--input-file="warehouse/locale/messages.pot" \
+		--output-file="warehouse/locale/$(L)/LC_MESSAGES/messages.po" \
+		--locale="$(L)"
+
+compile-po:
+	cd ../../; \
+	pybabel compile \
+		--input-file="warehouse/locale/$(L)/LC_MESSAGES/messages.po" \
+		--directory="warehouse/locale/" \
+		--locale="$(L)"
+
+build-mos: compile-pot
+	cd ../../; \
+	$(eval LOCALES := $(shell python -c "from warehouse.i18n import KNOWN_LOCALES; print(' '.join(set(KNOWN_LOCALES)-{'en'}))"))
+	for LOCALE in $(LOCALES) ; do \
```

Comment:
I'm somehow surprised that Make sees those 3 lines as a single block ? or is it that complil-po doensn'y actually need to be launched from 2 levels above ?